po/af.po

```diff
@@ -7,7 +7,7 @@
 msgid ""
 msgstr ""
 "Project-Id-Version: MusicBrainz\n"
-"Report-Msgid-Bugs-To: http://tickets.musicbrainz.org/\n"
+"Report-Msgid-Bugs-To: https://tickets.metabrainz.org/\n"
```

Comment:
I already commented on this in the PR description. :) I'd rather not mess with the commit anymore, and if the changes get wiped by the next Transifex pull, then that's fine too.